ansible/inventory/group_vars/operator-pipeline-integration-tests.yml

```diff
@@ -6,6 +6,7 @@ operator_pipeline_webhook_secret: ../../vaults/common/github-webhook-secret-prep
 kerberos_keytab: ../../vaults/common/nonprod-operatorpipelines.keytab
 
 operator_pipeline_image_tag: "v1.0.8"
+operator_pipeline_release_namespace: redhat-isv-operators-dev
```

Comment:
This one is a little confusing but I decided to match it to the `env` value above. @Allda do you know why we use dev here? The E2E-CI workflow uses `integration-tests` when setting up a custom environment but then invokes all the pipelines with `--param env=stage`.

Reply:
I'll try switching this to stage in a followup PR. I'm not certain it'll work as is but I'll hopefully be able to verify this after merging.